apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -36,6 +36,44 @@ namespace Google.Cloud.Spanner.V1
     /// </summary>
     public sealed partial class SessionPool
     {
+        /// <summary>
+        /// This struct contains information required to identify the database sessions.
+        /// It will be used as key in <see cref="_targetedPools"/>.
+        /// </summary>
+        internal readonly struct SessionPoolInfo
```

Comment:
As noted earlier, I'd make this public. But I'd rename it to something involving "key" (and it's not just SessionPoolKey, as it's the key of a pool *within* a session pool). It's not really information about the session pool itself.